internal/handlers/organization.go

```diff
@@ -541,12 +541,19 @@ func (api *API) DeleteOrganization(c *gin.Context) {
 	}
 
 	orgCIDR := org.IpCidr
+	orgCIDRV6 := org.IpCidrV6
 
 	if orgCIDR != "" {
 		if err := api.ipam.ReleasePrefix(c.Request.Context(), ipamNamespace, orgCIDR); err != nil {
 			c.JSON(http.StatusInternalServerError, models.NewApiInternalError(fmt.Errorf("failed to release ipam organization prefix: %w", err)))
 			return
 		}
 	}
+	if orgCIDRV6 != "" {
```

Comment:
This and the previous IF statements should also be in an `if org.PrivateCidr  { .... }` block.  Otherwise, we may try to delete the shared cidrs.